website/content/resources/newsletters/2024-04-25-ProjectNewsletter.md

```diff
@@ -47,4 +47,4 @@ We’re extending the window for Term 2 project ideas. You’ve now have until M
 
 ## Contribute!
 Want to contribute to the next newsletter? 
-[Contribute Today!](projects@cncf.io)
\ No newline at end of file
+[Contribute Today!](projects@cncf.io)
```

Comment:
I made this change via the GitHub UI..must have automatically added/removed a trailing new line. Lmk if I should change